haskell/ghc_bindist.bzl

```diff
@@ -301,8 +289,21 @@ _ghc_bindist = repository_rule(
             default = [],
             doc = "Sequence of commands to be applied after patches are applied.",
         ),
-        "locale": attr.string(
-            mandatory = False,
+        "patch_tool": attr.string(
+            default = "patch",
+            doc = "The patch(1) utility to use.",
+        ),
+        "patches": attr.label_list(
+            default = [],
+            doc =
+                "A list of files that are to be applied as patches afer " +
+                "extracting the archive.",
+        ),
+        "repl_ghci_args": attr.string_list(),
+        "target": attr.string(),
+        "version": attr.string(
+            default = _GHC_DEFAULT_VERSION,
+            doc = "The desired GHC version",
```

Comment:
Are these changes done by buildifier? Are we missing an option here that enforces sorting the dict keys?

Reply:
I reverted the unintended formatting changes for now.